src/main/java/bio/terra/service/dataset/AssetSpecification.java

```diff
@@ -73,4 +74,13 @@ public AssetTable getAssetTableByName(String tableName) {
         .findFirst()
         .orElseThrow();
   }
+
+  public boolean isFollowStrictDirection() {
+    return followStrictDirection;
+  }
+
+  public AssetSpecification setFollowStrictDirection(boolean followStrictDirection) {
```

Comment:
Since this a fluent-style method the convention is to not use `set`
```suggestion
  public AssetSpecification followStrictDirection(boolean followStrictDirection) {
```